### PR TITLE
tests/provider: Update hardcoded AZs (DocDB cluster instance)

### PR DESCRIPTION
**Pull request by YakDriver**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #12995 

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
NONE
```

Output from acceptance testing (GovCloud partition):

```
--- PASS: TestAccAWSDocDBClusterInstance_basic (541.30s)
```

Output from acceptance testing (commercial/standard partition):

```
Kommer snart...
```

### CODE REVIEW

File: aws/resource_aws_docdb_cluster_instance_test.go [hunk 1]
```diff
@@ -286,84 +291,107 @@ func testAccCheckAWSDocDBClusterInstanceExists(n string, v *docdb.DBInstance) re
...
+data "aws_docdb_orderable_db_instance" "test" {
+  engine                     = "docdb"
+  preferred_instance_classes = ["db.t3.medium", "db.r4.large", "db.r5.large", "db.r5.xlarge"]
 }
 
-func testAccAWSDocDBClusterInstanceConfig_namePrefix(n int) string {
-	return fmt.Sprintf(`
 resource "aws_docdb_cluster_instance" "test" {
-  identifier_prefix  = "tf-cluster-instance-"
+  identifier_prefix  = "tf-acc-docdb-"
```
Comment (bflad) [inline aws/resource_aws_docdb_cluster_instance_test.go:388]: Looks like just this small little thing:

```
=== CONT  TestAccAWSDocDBClusterInstance_namePrefix
TestAccAWSDocDBClusterInstance_namePrefix: resource_aws_docdb_cluster_instance_test.go:104: Check 4/4 error: aws_docdb_cluster_instance.test: Attribute 'identifier' didn't match "^tf-acc-test-", got "tf-acc-docdb-20200910175222397800000001"
--- FAIL: TestAccAWSDocDBClusterInstance_namePrefix (472.19s)
```

**Review by bflad — CHANGES_REQUESTED** [review]
Otherwise looking good. 👍

File: aws/resource_aws_docdb_cluster_instance_test.go (hunk 1 above)
Comment (YakDriver) [reply aws/resource_aws_docdb_cluster_instance_test.go:388]: Fixed.

**Comment by YakDriver** [comment]
Errant test:

```
--- PASS: TestAccAWSDocDBClusterInstance_namePrefix (666.01s)
```

**Review by bflad — APPROVED** [review]
Looks good 🚀 

Output from acceptance testing in AWS Commercial:

```
--- PASS: TestAccAWSDocDBClusterInstance_generatedName (498.41s)
--- PASS: TestAccAWSDocDBClusterInstance_az (579.42s)
--- PASS: TestAccAWSDocDBClusterInstance_disappears (605.85s)
--- PASS: TestAccAWSDocDBClusterInstance_namePrefix (611.02s)
--- PASS: TestAccAWSDocDBClusterInstance_kmsKey (627.32s)
--- PASS: TestAccAWSDocDBClusterInstance_basic (660.49s)
```

Output from acceptance testing in AWS GovCloud (US):

```
--- PASS: TestAccAWSDocDBClusterInstance_generatedName (491.53s)
--- PASS: TestAccAWSDocDBClusterInstance_disappears (524.09s)
--- PASS: TestAccAWSDocDBClusterInstance_kmsKey (621.14s)
--- PASS: TestAccAWSDocDBClusterInstance_basic (628.17s)
--- PASS: TestAccAWSDocDBClusterInstance_namePrefix (653.41s)
--- PASS: TestAccAWSDocDBClusterInstance_az (661.07s)
```

**Pull request merged**